examples/third_party/python/BUILD.python.bazel

```diff
@@ -0,0 +1,35 @@
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "configure_make")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "all_srcs",
+    srcs = glob(["**"]),
+)
+
+configure_make(
+    name = "python",
+    configure_options = [
+        "--with-openssl=$EXT_BUILD_DEPS/openssl",
+        "--with-zlib=$EXT_BUILD_DEPS/zlib",
+    ],
+    env = {"CFLAGS": "-Dredacted='\\\"redacted\\\"'"},
+    # rules_foreign_cc defaults the install_prefix to "python". This conflicts with the "python" executable that is generated.
+    install_prefix = "py_install",
+    lib_source = ":all_srcs",
+    out_binaries = [
+        "python3",
+        "pip3",
+    ],
+    out_data_dirs = ["lib"],
```

Comment:
What's the content of `lib`? I do feel this should be an explicit list.

Reply:
There are 8624 files in the lib folder. `python` needs access to these files at runtime

Reply:
owch, yeah, I guess directories are a necessity.